openshift/e2e-common.sh

```diff
@@ -37,11 +37,11 @@ if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
 elif [ -n "$DOCKER_REPO_OVERRIDE" ]; then
   readonly TEST_IMAGE_TEMPLATE="${DOCKER_REPO_OVERRIDE}/{{.Name}}"
 elif [ -n "$BRANCH" ]; then
-  readonly TEST_IMAGE_TEMPLATE="registry.svc.ci.openshift.org/openshift/${BRANCH}:knative-client-test-{{.Name}}"
+  readonly TEST_IMAGE_TEMPLATE="registry.ci.openshift.org/openshift/${BRANCH}:knative-client-test-{{.Name}}"
```

Comment:
You should prefer to pull this in with `base_images`:


```yaml
base_images:
  client-test:
    namespace: openshift
    name: 4.6
    tag: knative-client-test-whatever
```

Then, ask the system for where this image lives using `dependencies`: https://docs.ci.openshift.org/docs/architecture/ci-operator/#referring-to-images-in-tests

This will ensure that:

 - changes to this input image will invalidate the build cache that might otherwise cache this for you
 - images will be served from "local" registries to whichever server ends up running this, costing everyone less in network egress and being more efficient for you

Reply:
Thanks, I'll look into it.